packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -75,9 +75,10 @@ export function authorizationServiceBuilder(
     }): Promise<{ client: Client; showUsers: boolean }> {
       logger.info(`Retrieving Client ${clientId}`);
```

Comment:
since this PR is about `addUser`, are the changes in the `getClientById` expected?